src/main/java/com/jpa/entity/Developer.java

```diff
@@ -0,0 +1,46 @@
+package com.jpa.entity;
+
+import com.jpa.listeners.DeveloperListener;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Table;
+import javax.persistence.ManyToMany;
+import javax.persistence.CascadeType;
+import javax.persistence.FetchType;
+import javax.persistence.JoinTable;
+import javax.persistence.JoinColumn;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@EntityListeners(DeveloperListener.class)
+@Table(name = "developers")
+public class Developer extends AbstractIdentifiableObject{
+
+    @Column
+    private String name;
+
+    @Column
+    private Integer age;
+
+    @Column(name = "sex")
+    private boolean isMale;
```

Comment:
Is better not to use primitive. 
And in this case better even use enum or class object.
https://stackoverflow.com/questions/7506802/using-wrapper-integer-class-or-int-primitive-in-hibernate-mapping